pkg/catalog/render.go

```diff
@@ -0,0 +1,26 @@
+package catalog
+
+import (
+	"bytes"
+	"encoding/json"
+	"html/template"
+)
+
+func renderHTML(examples interface{}) ([]byte, error) {
+	exampleTmpl := template.New("example")
+	var contents bytes.Buffer
+	_, err := exampleTmpl.Parse(catalogTemplate)
+	if err != nil {
+		return nil, err
+	}
+	err = exampleTmpl.Execute(&contents, examples)
+	return contents.Bytes(), err
+}
+
+func renderJSON(examples interface{}) ([]byte, error) {
+	j, err := json.Marshal(examples)
+	if err != nil {
+		return nil, err
+	}
+	return j, err
+}
```

Comment:
nitpicky but why use this instead of just calling `json.Marshall` directly?

Reply:
yeah this function used to do a lot more